Formula/ibazel.rb

```diff
@@ -0,0 +1,66 @@
+class Ibazel < Formula
+  desc "Tools for building Bazel targets when source files change"
+  homepage "https://github.com/bazelbuild/bazel-watcher"
+  url "https://github.com/bazelbuild/bazel-watcher/archive/refs/tags/v0.20.0.tar.gz"
+  sha256 "7b0f98006b32d2ad5eee38eb9b363005968abf88d235e31fc6dd2a8c4336f33d"
+  license "Apache-2.0"
+
+  depends_on "bazel" => [:build, :test]
+  depends_on "go" => [:build, :test]
+  depends_on :macos
+
+  def install
+    inreplace ".bazelversion", "5.3.1", "5.3.2"
```

Comment:
Why?

Reply:
there is some build issue

```
ERROR: The project you're trying to build requires Bazel 5.3.1 (specified in /private/tmp/ibazel-20221117-33447-o7pxmb/bazel-watcher-0.20.0/.bazelversion), but it wasn't found in /opt/homebrew/Cellar/bazel/5.3.2/libexec/bin.
```

Should be covered by https://github.com/bazelbuild/bazel-watcher/pull/556

Reply:
We should just wait for the next release then

Reply:
yeah, that works for me.